src/components/structures/ViewSource.tsx

```diff
@@ -77,9 +76,13 @@ export default class ViewSource extends React.Component<IProps, IState> {
                         <summary>
                             <span className="mx_ViewSource_heading">{_t("Decrypted event source")}</span>
                         </summary>
-                        <CopyableText getTextToCopy={copyDecryptedFunc}>
-                            <SyntaxHighlight language="json">{stringify(decryptedEventSource)}</SyntaxHighlight>
-                        </CopyableText>
+                        {decryptedEventSource ? (
+                            <CopyableText getTextToCopy={copyDecryptedFunc}>
+                                <SyntaxHighlight language="json">{stringify(decryptedEventSource)}</SyntaxHighlight>
+                            </CopyableText>
+                        ) : (
+                            <div>{_t("Decrypted source unavailable - event could not be decrypted")}</div>
```

Comment:
```suggestion
                            <div>{_t("Decrypted source unavailable")}</div>
```

Oh, you're totally right, this is what it should be ^, right?

Reply:
Sure, sounds good to me!